webcompat/static/js/lib/issues.js

```diff
@@ -436,6 +438,24 @@ issues.MainView = Backbone.View.extend({
         wcEvents.trigger('flash:error', {message: msg, timeout: 4000});
       }
     });
+    var i;
+    for (i = 2; i <= this.lastPageNumber; i++) {
+      this.commentPage.page = i;
```

Comment:
If this property is only used in the `fetchPage` call below, can we just do `{page: i...}`?